dist/logger.manager.js

```diff
@@ -63,7 +63,7 @@ var LoggerManager = (function () {
         debug_logger_1.DebugLogger.d("starting log-manager");
         this.logBulkObs$ = rxjs_1.Observable.create(function (observer) { return _this.addLogStream = observer; })
             .do(function (log) { return log_1.Log.fillDefaultValidValues(log); })
-            .do(function (log) { return _this.bufferSize >= constants_1.Constants.MAX_LOG_BUFFER_SIZE ? debug_logger_1.DebugLogger.d("max logs exceeded, dropping log") : null; })
+            .do(function (log) { return _this.bufferSize >= constants_1.Constants.MAX_LOG_BUFFER_SIZE ? console.log("max logs exceeded, dropping log") : null; })
```

Comment:
Console log is sync process and has bad performance, I understand that you willing make it easier to debug but the SDK is running in customers services we cannot allow it to impact the performance

Reply:
I know this, but Roy asked me to show this message in any case.

Reply:
can't we use the debug_logger or another instance of it to write to the stdout? console.log is not acceptable

Reply:
@amnons77 Dropping logs is not something that we should print only in debug
we should always have a log that shows if we have such an issue otherwise customer will never know he is dropping logs